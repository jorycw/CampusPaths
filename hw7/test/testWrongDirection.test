# Find a path with Dijkstra's algorithm when the path s
# between the two nodes are not the correct direction

CreateGraph graph

AddNode graph a
AddNode graph b
AddNode graph c

AddEdge graph c b 1
AddEdge graph b a 1

FindPath graph a c