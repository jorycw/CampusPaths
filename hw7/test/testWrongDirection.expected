# Find a path with Dijkstra's algorithm when the path s
# between the two nodes are not the correct direction

created graph graph

added node a to graph
added node b to graph
added node c to graph

added edge 1.000 from c to b in graph
added edge 1.000 from b to a in graph

path from a to c:
no path found